bedrock/base/templates/macros.html

```diff
@@ -474,27 +474,3 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     </footer>
   </section>
 {%- endmacro %}
-
-
-{% macro google_custom_search(search_type, type_class, default_term, term_list) %}
```

Comment:
Hmm... I don't remember that. Cached in what way? We should be able to remove any macro as long as it's no longer used by any templates.